blackbox/docs/editions/enterprise.rst

```diff
@@ -33,6 +32,12 @@ enterprise features are enabled:
 - `The CrateDB admin UI`_: `shards browser`_, `monitoring overview`_,
   `privileges browser`_
 
+
+.. note::
+
+   It is also possible to build a :ref:`community-edition` which won't contain
+   these features but which can be used without licensing restrictions.
+
 .. _enterprise_trial:
 
 Trial
```

Comment:
Note that this isn't updated yet because the logic change hasn't been done/merged yet

Reply:
makes sense now that you mentioned - I was about to ask :)